style.css

```diff
@@ -313,3 +326,63 @@ button {
   align-items: center;
   gap: 10px;
 }
+
+.contact_form {
+  background-color: #fff;
+  margin: 0 auto;
+  padding-top: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#contact_header {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-weight: 700px;
+  margin-bottom: 80px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact_form input,
+textarea {
+  width: 327px;
+  border: none;
+  color: #979493;
+  margin-top: 30px;
+}
+
+#button_label {
+  margin: 80px auto;
+  width: 131px;
+  height: 48px;
+  font-size: 10px;
+  background-color: #ff6b00;
+  font-weight: bolder;
+}
+
+#under_input {
+  margin-bottom: 10px;
+}
+
+#above_footer {
+  margin-top: 110px;
+  margin-bottom: -100px;
+  height: 1px;
+  color: #dfe1e6;
+}
+
+#user_comment {
+  background-color: #fbf8f7;
+}
+
+textarea {
+  margin-top: 30px;
+}
```

Comment:
- [ ] You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). At the Minimum mobile screen size of 375px, the contact section should be responsive, so Kindly work on it.

Your Contact form at  Minimum mobile screen size of 375px Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/192598074-ef83051a-93b0-404b-bb30-e7623aaaced6.PNG)

The Figma design  at 375px Look Likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/192598295-061c874a-58a6-4d3b-8ec1-38e9d72e546d.PNG)